VisualStudio.gitignore

```diff
@@ -51,6 +51,13 @@ project.fragment.lock.json
 artifacts/
 **/Properties/launchSettings.json
 
+# StyleCop
+StyleCopReport.xml
```

Comment:
Any documentations or discussions to explain the context behind this change?

Reply:
Unfortunately I can't found out any documents described the output files...
But this file appears very often in our projects, so I propose add it for now and add/change in future, when will have more info.